rosdep/python.yaml

```diff
@@ -6709,6 +6709,19 @@ python3-mypy:
     '7': null
     '8': null
   ubuntu: [python3-mypy]
+python3-natsort:
+  debian: [python3-natsort]
+  fedora: [python3-natsort]
+  opensuse: [python3-natsort]
+  rhel: [python3-natsort]
+  ubuntu: [python3-natsort]
+python3-natsort-pip:
```

Comment:
Please remove the pip rule. The natively packaged version is the version we will select if it's available.